docs/metrics.md

```diff
@@ -4,7 +4,7 @@ This document contains all the prometheus metrics exposed by a charon node.
 
 All metrics contain the following labels, so they are omitted from the table below:
 - `cluster_hash`: The cluster lock hash uniquely identifying the cluster.
-- `clustter_name`: The cluster lock name.
```

Comment:
@futreall `metrics.md` is auto-generated from the source code.

To resolve the build failure, you will need to update `markdown_internal_test.go` line 87: https://github.com/ObolNetwork/charon/blob/9151aa051371025a89d75c60f79fbefa0bb71032/cmd/markdown_internal_test.go#L87

This will make the build passing as the change will match the changes you made to `metrics.md`.

Thank you!